blueprints/ember-cp-validations/index.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   afterInstall: function() {
     return RSVP.all([
       this.addAddonToProject('ember-lodash', '0.0.5'),
-      this.addAddonToProject('ember-moment', '3.2.1')
+      this.addBowerPackageToProject('moment', '>= 2.8.0'),
```

Comment:
This was causing some failures with ember ^2.0 but now that I look at it.... let me revert back and upgrade ember-moment to the latest version.